<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentClass</class>
 <widget class="QMainWindow" name="AddStudentClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddStudent</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="dataLine" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>541</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="snoLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLine">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="sdeptBox">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>120</y>
       <width>68</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>信息科学与工程学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>心理学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="submitBtn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sexBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>68</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
